types/store.go

```diff
@@ -256,3 +256,10 @@ func PrefixEndBytes(prefix []byte) []byte {
 	}
 	return end
 }
+
+//----------------------------------------
+
+// key-value result for iterator queries
+type KV struct {
```

Comment:
cmn.KVPair? https://github.com/tendermint/tmlibs/blob/master/common/kvpair.go#L14

Reply:
awesome will update for this